packages/wallet-connect-sdk-react/package.json

```diff
@@ -28,20 +28,14 @@
     "build": "microbundle-crl --no-compress --format modern,cjs",
```

Comment:
It's better to change the version of this package as well (react), to the same number of the core